metricbeat/modules.d/system.yml

```diff
@@ -1,14 +1,20 @@
 - module: system
+  period: 10s
+  metricsets:
+    - cpu
+    - load
+    - memory
+    - network
+    - process
+    - process_summary
+    #- core
+    #- diskio
+    #- socket
+  processes: ['.*']
   process.include_top_n:
     by_cpu: 5      # include top 5 processes by CPU
     by_memory: 5   # include top 5 processes by memory
 
-#- module: system
-#  metricsets:
-#    - core
-#    - diskio
-#    - socket
-
 - module: system
   period: 1m
   metricsets:
```

Comment:
I think it might make sense to use the` ignore_types` option instead of the filter. Or maybe these mounts are filtered by default now?

Reply:
That's a good question, @jsoriano do your recent changes affect this?

Reply:
Oh sorry, I forgot this question. My latest changes filter by default the filesystem types marked as `nodev` in `/proc/filesystems`, and remove duplications caused by bind mounts and similar. Probably this is enough to have meaningful events by default without needing these processors.